manifests/a/Adobe/Acrobat/Reader/64-bit/21.007.20099/Adobe.Acrobat.Reader.64-bit.locale.en-US.yaml

```
@@ -0,0 +1,27 @@
+# Created with YamlCreate.ps1 v2.0.0 $debug=AUSU.5-1-19041-1237
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Adobe.Acrobat.Reader.64-bit
+PackageVersion: 21.007.20099
+PackageLocale: en-US
+Publisher: Adobe
+PublisherUrl: https://www.adobe.com
+PublisherSupportUrl: https://helpx.adobe.com/support.html
+PrivacyUrl: https://www.adobe.com/privacy.html
+Author: Adobe Systems Incorporated
+PackageName: Adobe Acrobat DC (64-bit)
+PackageUrl: https://acrobat.adobe.com/us/en/acrobat/pdf-reader.html
+License: Proprietary, Freeware
+LicenseUrl: https://www.adobe.com/content/dam/cc/en/legal/licenses-terms/pdf/Reader-EULA-en_US-20181207.pdf
+Copyright: Copyright (c) Adobe
+CopyrightUrl: https://www.adobe.com/content/dam/cc/en/legal/licenses-terms/pdf/Reader-EULA-en_US-20181207.pdf
+ShortDescription: The free global standard for reliably viewing, printing, signing, and commenting on PDF documents.
+Description: Adobe Acrobat Reader DC software is the free, trusted global standard for viewing, printing, signing, sharing, and annotating PDFs. It's the only PDF viewer that can open and interact with all types of PDF content – including forms and multimedia.
+Moniker: acrobatreader
```

```suggestion
Moniker: adobe-reader
```

People usually refer to this app in speech as Adobe Reader. Unless someone has an objection, the moniker should probably be changed.

---

I was simply going with what was originally in the 32-bit version; I don't have an objection to changing it, but addiitonal opinions would be appreciated

---

We shouldn't have a moniker pointing to two different PackageIdentifiers, that means you can't use it to install the package. So maybe `adobe-reader-x64`?